packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -72,15 +78,17 @@ export class W3mAccountDefaultWidget extends LitElement {
       throw new Error('w3m-account-view: No account provided')
     }
 
+    const shouldShowMultiAccount =
+      ChainController.state.activeChain === ConstantsUtil.CHAIN.EVM && this.allAccounts.length > 1
+
     return html`<wui-flex
         flexDirection="column"
         .padding=${['0', 'xl', 'm', 'xl'] as const}
         alignItems="center"
         gap="l"
+        data-testid=${shouldShowMultiAccount ? 'multi-account-widget' : 'single-account-widget'}
```

Comment:
why not move the id to the corresponding template instead of conditionally adding it here?

Reply:
Yeah let me do that